includes/embeds/class-amp-gallery-embed.php

```diff
@@ -224,4 +234,21 @@ public function render( $args ) {
 			implode( PHP_EOL, $images )
 		);
 	}
+
+	/**
+	 * Enqueues the Gallery block styling.
+	 *
+	 * It would be better to enqueue this in AMP_Gallery_Block_Sanitizer,
+	 * but by the time that runs, it's too late to call wp_enqueue_style().
+	 *
+	 * @return void
+	 */
+	public function enqueue_styles() {
+		wp_enqueue_style(
```

Comment:
Not a big deal, but why have the external CSS file here but not in `add_twentynineteen_image_styles`? I'm fine with having such a small amount of CSS just being inline in the PHP.

Reply:
Looks fine. And using an anonymous function works well for me.

Reply:
Thanks, how does https://github.com/Automattic/amp-wp/pull/1627/commits/6b7d305ced47864ecf0c898338889e4857da7cc3 look?